net/badvpn/Makefile

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2009 Ambroz Bizjak
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=badvpn
+PKG_VERSION:=1.999.130
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=HEAD
```

Comment:
Use a tag och commit here, pretty sure this will break as soon as anything is committed to the repo.